dockstore-webservice/src/main/java/io/dockstore/webservice/languages/CWLHandler.java

```diff
@@ -753,8 +753,12 @@ public VersionTypeValidation validateWorkflowSet(Set<SourceFile> sourcefiles, St
             isValid = false;
         }
 
-        validationMessageObject.put(primaryDescriptorFilePath, validationMessage.toString());
-        return new VersionTypeValidation(isValid, validationMessageObject);
+        if (isValid) {
+            return new VersionTypeValidation(true, new HashMap<>());
```

Comment:
You can do `Collections.emptyMap()` instead of creating a new object, unless you're planning to add stuff to the map later. The tiniest of micro-optimizations. :)